<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicDownloadBatchWidget</class>
 <widget class="QWidget" name="MusicDownloadBatchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>下载窗口</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>556</width>
     <height>390</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>下载窗口</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>538</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>556</width>
     <height>365</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{color:#666666;background:white;}
QLabel{background:transparent;}</string>
   </property>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>318</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>批量下载</string>
    </property>
   </widget>
   <widget class="QLabel" name="songLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>歌曲名</string>
    </property>
   </widget>
   <widget class="QLabel" name="singerLabel">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>10</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>歌手</string>
    </property>
   </widget>
   <widget class="QLabel" name="qualityLabel">
    <property name="geometry">
     <rect>
      <x>313</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>品质</string>
    </property>
   </widget>
   <widget class="MusicDownloadBatchTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>35</y>
      <width>540</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="songCountLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>295</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:bold;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="qualityBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>7</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicDownloadBatchTableWidget</class>
   <extends>QTableWidget</extends>
   <header>musicdownloadbatchwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
